plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabGroupEnsureExistsAction.ts

```diff
@@ -52,6 +53,11 @@ export const createGitlabGroupEnsureExistsAction = (options: {
       }),
     },
     async handler(ctx) {
+      if (ctx.isDryRun) {
+        ctx.output('groupId', 42);
```

Comment:
Not quite able to formulate a question regarding this specific choice. It'll do 👍